src/database/import.ts

```diff
@@ -31,15 +28,17 @@ type ChunkedData = {
  */
 export default class DatabaseImporter {
   private client: Client;
-  private limit = pLimit(CONCURRENCY_LIMIT);
+  private limit: pLimit.Limit;
 
   constructor(
     private dbUrl: URL,
     private inStream: stream.Readable,
     private dataPath: string,
-    private chunkSize = MAX_CHUNK_SIZE
+    private chunkBytes: number,
+    concurrency: number
```

Comment:
Any reason this can't be private as well?

Reply:
It can be; I just don't need it outside of the constructor.